docker/Dockerfile

```diff
@@ -51,20 +62,21 @@ RUN ln -s /user-data /home/$DOCKER_USER/data
 #RUN curl -LO https://raw.githubusercontent.com/broadinstitute/viral-ngs-deploy/master/easy-deploy-script/easy-deploy-viral-ngs.sh
 USER root
 ADD ./easy-deploy-script/easy-deploy-viral-ngs.sh /home/$DOCKER_USER/easy-deploy-viral-ngs.sh
-RUN chmod +x ./easy-deploy-viral-ngs.sh
-RUN chown $DOCKER_USER:$DOCKER_USER ./easy-deploy-viral-ngs.sh
+RUN chmod +x ./easy-deploy-viral-ngs.sh && \
+    chown $DOCKER_USER:$DOCKER_USER ./easy-deploy-viral-ngs.sh
 USER $DOCKER_USER
 ENV VIRAL_NGS_VERSION $VIRAL_NGS_VERSION
+
 RUN echo "Running easy install with viral-ngs version $VIRAL_NGS_VERSION"
-RUN bash -c 'echo $(if [ "$VIRAL_NGS_VERSION" == "" ]; then echo "./easy-deploy-viral-ngs.sh setup"; else echo "./easy-deploy-viral-ngs.sh setup --viral-ngs-version $VIRAL_NGS_VERSION" ; fi)'
```

Comment:
Let's keep the echo line in the file; it is help for debugging when things go wrong with our automated post-deploy Travis API call.